doc/source/user_guide/label_guide.rst

```diff
@@ -97,22 +103,25 @@ Sorting variable names
 Sorting coordinate values
 .........................
 
-We may also want to sort the schools by their mean.
-To do so we first have to get the means of each school:
+To sort coordinate values you have to define the order, store it, and use the result to sort the coordinate values. 
+
+Example: Sorting the schools by mean
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1) Locate the means of each school by using the following command:
 
 .. ipython::
 
     In [1]: school_means = schools.posterior["theta"].mean(("chain", "draw"))
        ...: school_means
 
-We can then use this DataArray result to sort the coordinate values for ``theta``.
-Again we have two alternatives:
+2) You can use the DataArray result to sort the coordinate values for ``theta``.
+There are two ways of sorting:
 
 .. tabbed:: ArviZ args
 
-  Here the first step is to sort the coordinate values so we can pass them as `coords` argument and
-  choose the order of the rows.
-  If we want to manually sort the schools, `sorted_schools` can be defined straight away as a list
+  Sort the coordinate values to pass them as a `coords` argument and choose the order of the rows.
+  To manually sort the schools, `sorted_schools`, define sorted_schools as a list.
```

Comment:
To keep the example only about sorting by mean, I propose to move this above (I am commenting later in time on the above section in hopes that github will preserve the chronology).

```suggestion
```